react/ConditionLayoutProduct.tsx

```diff
@@ -70,6 +72,17 @@ export const HANDLERS: Handlers<SubjectValues, SubjectArgs> = {
 
     return specification.values.includes(String(args?.value))
   },
+  isProductAvailable({ values }) {
+    const { isProductAvailable: sellers } = values
+
+    const sellersWithProductAvailable = sellers?.filter(
+      (seller) => seller.commertialOffer.AvailableQuantity > 0
+    )
+
+    return (
+      !!sellersWithProductAvailable && sellersWithProductAvailable.length > 0
+    )
```

Comment:
What do you think of it?

```suggestion
    const isAvailable = sellers?.some(
      (seller) => seller.commertialOffer.AvailableQuantity > 0
    )

    return isAvailable
```

Reply:
It's good idea. I'm goint to do it right now.

Reply:
@kaisermann It's done!